src/RevPrefetcher.cc

```diff
@@ -38,6 +39,10 @@ bool RevPrefetcher::IsAvail(uint64_t Addr){
         return false;
       }
 
+      if( Addr == 0x4bd38ull ){
+        std::cout << "not stalled" << std::endl;
+      }
+
```

Comment:
remove address check

Reply:
resolved in https://github.com/tactcomplabs/rev/commit/e9a5df3d3de81136677d389638c82e5ced6b65c9